### PR TITLE
[PO-8] Model leader selection

### PR DESCRIPTION
**Pull request by mirkoAlic**

### CODE REVIEW

File: src/main/scala/io/iohk/praos/App.scala [hunk 1]
```diff
@@ -7,19 +7,23 @@ object App {
...
     val env: Environment = Environment(
       stakeDistribution,
       epochLength = 10,
       slotDurationInMilliseconds = 3000,
-      timeProvider = new TimeProvider(initialTime = 0)
+      timeProvider = new TimeProvider(initialTime = 0),
+      activeSlotCoefficient = 0.70
     )
-    val slotInEpochCalculator = new SlotInEpochCalculator(
+    val slotInEpochCalculator = SlotInEpochCalculator(
```
Comment (AlanVerbner) [inline src/main/scala/io/iohk/praos/App.scala:26]: :+1:

File: src/main/scala/io/iohk/praos/domain/ElectionManager.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  def isStakeHolderLeader(stakeholder: StakeHolder, genesis: GenesisBlock, slotInEpoch: SlotInEpoch): Option[VrfProof] = {
+    // TODO 1: Improve this step with real concatenation and convert slot number into a PRNG seed.
+    // TODO 2: Check if it is necessary concat another Nonce.
+    val randomSeed: RandomValue = genesis.genesisNonce + slotInEpoch.slotNumber
+    val vrfProof = vrf.prove(stakeholder.privateKey, randomSeed)
+
+    val probLeader = this.probLeader(RelativeStakeCalculator.calculate(stakeholder.publicKey, genesis.genesisDistribution))
+    val probLeaderThreshold = this.probLeaderThreshold(VrfProof.randomLength, probLeader)
+    println("probLeaderThreshold", probLeaderThreshold, " vs random", abs(vrfProof.random))
```
Comment (AlanVerbner) [inline src/main/scala/io/iohk/praos/domain/ElectionManager.scala]: :thinking:

Comment (AlanVerbner) [reply src/main/scala/io/iohk/praos/domain/ElectionManager.scala]: Let's use logback to log stuff

**Review by ntallar — APPROVED** [review]
LGTM!

**Pull request merged**